sys/linux/socket_netlink_generic_gtp.txt

```diff
@@ -31,17 +32,24 @@ gtp_genl_policy [
 	GTPA_NET_NS_FD		nlattr[GTPA_NET_NS_FD, fd_namespace]
 	GTPA_I_TEI		nlattr[GTPA_I_TEI, int32[0:4]]
 	GTPA_O_TEI		nlattr[GTPA_O_TEI, int32[0:4]]
+	GTPA_PEER_ADDR6		nlattr[GTPA_PEER_ADDR6, ipv6_addr]
+	GTPA_MS_ADDR6		nlattr[GTPA_MS_ADDR6, ipv6_addr]
+	GTPA_FAMILY		nlattr[GTPA_FAMILY, int8[socket_domain]]
 ] [varlen]
 
 gtp_policy {
-	IFLA_GTP_FD0	nlattr[IFLA_GTP_FD0, sock_udp]
-	IFLA_GTP_FD1	nlattr[IFLA_GTP_FD1, sock_udp]
-	var		array[gtp_policy$var]
+	var	array[gtp_policy$var]
```

Comment:
I think it's better to fix the use of `array[gtp_policy]` in socket_netlink_route.txt:
```
# This is not an array[gtp_policy] intentionally, gtp_policy is a struct.
	gtp		ifla_info_policy$t["gtp", IFLA_INFO_DATA, gtp_policy]
	hsr		ifla_info_policy$t["hsr", IFLA_INFO_DATA, array[hsr_policy]]
	xfrm		ifla_info_policy$t["xfrm", IFLA_INFO_DATA, array[xfrmi_policy]]
```

It will be more consistent with other policies (hsr/xfrm), we won't need 2 separate types, and we can remove that comment as well.